platform/osx/app/mapboxgl-app.gypi

```diff
@@ -60,7 +60,7 @@
       
       'copies': [
         {
-          'destination': '<(PRODUCT_DIR)/$(FRAMEWORKS_FOLDER_PATH)',
+          'destination': '<(PRODUCT_DIR)/\${FRAMEWORKS_FOLDER_PATH}',
```

Comment:
This is incompatible with Xcode. The destination path becomes “\${FRAMEWORKS_FOLDER_PATH}”, and osxapp crashes on launch (when not launched through the debugger).